webapp/src/test/java/com/box/l10n/mojito/android/strings/AndroidStringDocumentMapperTest.java

```diff
@@ -782,4 +783,461 @@ private TextUnitDTO textUnitDTO(
 
     return textUnit;
   }
+
+  @Test
+  public void testReadFromSourceTextUnitsWithDuplicatedOnePluralsWithDifferentContent() {
```

Comment:
This would be the test when the one plural is modified:
```
<!-- %1$d: number of sections we have in the board 2 -->
    <plurals name="plural_homefeed_section_test">
        <item quantity="one">\u0020· %1$d white test section</item>
        <item quantity="other">\u0020· %1$d test sections</item>
    </plurals>
```

Reply:
Discussed offline but the generated Android xml file here is only populating the singular string in the plurals set for the duplicated one value.

```
<plurals name="my/path0.xml#@#name1">
<item quantity="one" tmTextUnitId="201">content2_one</item>
</plurals>
```

I suspect similar may be occurring for the other duplication too